kimchi/src/circuits/expr.rs

```diff
@@ -468,7 +488,10 @@ pub enum PolishToken<F> {
     Gamma,
     JointCombiner,
     EndoCoefficient,
-    Mds { row: usize, col: usize },
+    Mds {
+        row: usize,
+        col: usize,
+    },
```

Comment:
I'm amazed that `cargo fmt` is not complaining :P

Reply:
It did this automatically, it wasn't me..

Reply:
maybe the latest rust update